_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -67,17 +67,37 @@ This `div` element has a `letter-spacing` of `0.12em` which is equal to the reco
 </html>
 ```
 
+#### Passed Example 2
+
+This `p` element has a `letter-spacing` of `24px` specified via the style attribute, which is above to the recommended minimum given the specified font size is 16 pixels.
+
+```html
+<html>
+	<style>
+		body {
+			font-size: 16px;
+		}
+	</style>
+
+	<body>
+		<p style="letter-spacing: 24px;">
+			The toy brought back fond memories of being lost in the rain forest.
+		</p>
+	</body>
+</html>
+```
+
 ### Failed
 
 #### Failed Example 1
 
-This `div` element has a `letter-spacing` of `1.5px !important` (equals `0.09375em`) which is below to the recommended minimum, given the specified font size is 16 pixels.
+This `div` element has a `letter-spacing` of `1.5px !important` (equals `0.09375em`) which is below to the recommended minimum given the specified font size is `1em`.
```

Comment:
I guess the size of the `div` should also use relative units now that `font-size` does.

Reply:
I'll let @EmmaJP decide on that since she was the one who requested relative units in the first place (https://github.com/act-rules/act-rules.github.io/pull/1412#pullrequestreview-466875191)

Reply:
I think for this failed example it certainly demonstrates why those fixed units are a problem. I'm unsure if using different units for the letter-spacing and font-size muddies the waters or not. 

Certainly for an examples using the relative units, we should probably clarify that at default zoom 1em would be 16px.